src/StatusQ/Popups/StatusStackModal.qml

```diff
@@ -55,8 +55,6 @@ StatusModal {
     leftButtons: StatusRoundButton {
         id: backButton
         icon.name: "arrow-right"
-        icon.width: 20
-        icon.height: 16
         rotation: 180
```

Comment:
Just curious... don't we have a "arrow-left" icon? And secondly, maybe this "Back" button is worth a separate/reusable component?

Reply:
> Just curious... don't we have a "arrow-left" icon? 

We do by now: https://github.com/status-im/StatusQ/blob/master/src/assets/img/icons/arrow-left.svg

>  maybe this "Back" button is worth a separate/reusable component?

Yup